LondonBritishLibrary/orient/BLorient8819.xml

```diff
@@ -0,0 +1,279 @@
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 8819</idno>
+                        <altIdentifier><idno>Or. 8819</idno></altIdentifier>
+                        <altIdentifier><idno>Strelcyn 57</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="4r" to="78r"/> 
+                            <title ref="LIT6193HomiliaryPassions">Discourses on the Passion of our Lord Jesus Christ for Holy Week</title>  
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="4r"/> 
+                                <title ref="LIT6193HomiliaryPassions#Introduction"/>
+                                <note>Rhyming</note>
+                                <incipit xml:lang="gez">በስመ፡ እግዚአብሔር፡ ሥሉስ፡ ዘእንበለ፡ ቀዲም፡ ወተድኅሮ። በግጻዌሁ፡ ይትረአይ፡ ዘአስተዋሐደ፡ ምክሮ። በጠፈረ፡ በረድ፡ ወነድ፡
+                                    ዘሐነጸ፡ ማኅደሮ። ወበአርእስተ፡ ኪሩቤል፡ ርቱዓ፡ ዘአማዕዘነ፡ መንበሮ። </incipit>
+                            </msItem>
+                            <msItem xml:id="ms_i1.2">
+                                <locus from="6v"/> 
+                                <title ref="LIT6193HomiliaryPassions#Monday"/>
+                                <note>Indicated by <foreign xml:lang="gez">ዘሰኑይ፡ </foreign> on the upper margin of <locus target="#4r"/>.</note>
+                                <incipit xml:lang="gez">ናንቅሕ፡ ርእሰ፡ ሕሊናነ፡ ወናስተሐውስ፡ አስራወ፡ ልሳንነ፡ ንክሥት፡ አፉነ፡ ወናስተናብብ፡ ከናፍሪነ። ከመ፡ ንንግር፡ ዜና፡ ሕማማቲሁ፡ 
+                                    ለመድኅኒነ፡ እንዘ፡ ኃያል፡ ዘኢየሐምም፡</incipit>
+                            </msItem>
+                            <msItem xml:id="ms_i1.3">
+                                <locus from="17r"/> 
+                                <title ref="LIT6193HomiliaryPassions#HomilyTuesdayOther">Homily for Tuesday</title>
+                                <note>According to <bibl><ptr target="bm:Strelcyn1978BritishLibrary"/><citedRange unit="page">92</citedRange>
```

Comment:
I find the description of Strelcyn very unclear at this point. ዘሰኑይ is written in the upper margin (it refers to the text starting from 1A, not 1B); and as such this info would be for Varia; how does he define the days for parts B - H? One can only guess that he did that basing on similar notes in the margin, which he skipps in the description (except the firs one). I wonder if later it would be possible to come to the website of the British Library and check if this manuscript will be online, to ascertain that the notes referring to the days of the week do exist. Then they can be placed in varia. (The repeating passage "According to "Strelcyn....", up to et 116 could be deleted).